skyvern-frontend/src/util/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
nit: what does `cn` stand for? can we get a better function name?

Reply:
Stands for `classnames`. It's a simple string utility to easily merge css classnames, include / exclude them conditionally, deduplicate etc.

I think it's a convention of this particular library, but there is no harm in renaming it, and I agree it can be confusing. I'll change the name.

Reply:
On second thought, maybe I won't break the convention. Because when I use the CLI to add components, it will assume it uses the same `cn` name. It seems like people who use this library are used to that, and I wouldn't want to confuse any future collaborators. Maybe I'll come up with another way to work around this later.

Reply:
Got it. i'm okay with it as long as we're aligned with the convention